src/github/deployment.ts

```diff
@@ -69,11 +69,6 @@ export const processDeployment = async (
 	);
 
 	const result: DeploymentsResult = await jiraClient.deployment.submit(jiraPayload);
-	if (result.rejectedDeployments?.length) {
```

Comment:
Moved into the jira client. So that we can also capture the error logs for deployments in backfill.